usaspending_api/etl/management/commands/zero_missing_award_outlays.py

```diff
@@ -11,22 +11,37 @@
 
 MISSING_COVID_AWARD_SQL = """
```

Comment:
Based on our discussion, please add a DISCLAIMER above this SQL that states it is only good for FY2020, since it does not adequately account for multiple fiscal years. Further changes incoming.